docs/docs/hive.md

```diff
@@ -841,8 +842,6 @@ ALTER TABLE ice_t EXECUTE ROLLBACK(1111);
 ### Compaction
 
 Hive 4 supports full table compaction of Iceberg tables using the following commands:
-* Using the `ALTER TABLE ... COMPACT` syntax
```

Comment:
Is this just a removal?

Reply:
I removed this comment because the following SQL example contains the same comment.